src/utils/visualizer/visualizer-1.js

```diff
@@ -17,134 +17,133 @@ const setIdentity = require('gl-mat4/identity')
 
-    const controls = createOrbit({
-      element: canvas,
-      distanceBounds: [0.4, 3],
-      distance: 0.4
-    })
+    shader.bind()
+    shader.uniforms.iGlobalTime = time
+    shader.uniforms.radius = radius
+    shader.uniforms.audioTexture = 0
+    shader.uniforms.opacity = settings.opacity
+    shader.uniforms.useHue = settings.useHue
+
+    if (analyser) {
+      analyser.bindFrequencies(0)
+    }
 
-    const paths = newArray(segments).map(createSegment)
-    const lines = paths.map(path => {
-      return createLine(gl, shader, path)
+    lines.forEach((line, i, list) => {
+      line.color = colorVec
+      line.thickness = thickness
+      line.model = identity
+      line.view = camera.view
+      line.projection = camera.projection
+      line.aspect = width / height
+      line.miter = 0
+      shader.uniforms.index = i / (list.length - 1)
+      line.draw()
     })
+  })
 
-    let time = 0
-    app.on('tick', dt => {
-      time += Math.min(30, dt) / 1000
-
-      const width = gl.drawingBufferWidth
-      const height = gl.drawingBufferHeight
-
-      // set up our camera
-      camera.viewport[2] = width
-      camera.viewport[3] = height
-      controls.update(camera.position, camera.direction, camera.up)
-      camera.update()
-
-      gl.viewport(0, 0, width, height)
-      gl.clearColor(0, 0, 0, 1)
-      gl.clear(gl.COLOR_BUFFER_BIT | gl.DEPTH_BUFFER_BIT)
-
-      const size = Math.min(width, height) * 1.5
-      gl.disable(gl.DEPTH_TEST)
-      background.style({
-        color1: hexRgb(settings.gradient[0]),
-        color2: hexRgb(settings.gradient[1]),
-        scale: [ 1 / width * size, 1 / height * size ]
-      })
-      background.draw()
-
-      gl.disable(gl.DEPTH_TEST)
-      gl.enable(gl.BLEND)
-      if (settings.additive) gl.blendFunc(gl.ONE, gl.ONE)
-      else gl.blendFunc(gl.ONE, gl.ONE_MINUS_SRC_ALPHA)
-      gl.disable(gl.CULL_FACE)
-
-      shader.bind()
-      shader.uniforms.iGlobalTime = time
-      shader.uniforms.radius = radius
-      shader.uniforms.audioTexture = 0
-      shader.uniforms.opacity = settings.opacity
-
-      if (analyser) {
-        analyser.bindFrequencies(0)
-      }
-
-      lines.forEach((line, i, list) => {
-        line.color = colorVec
-        line.thickness = thickness
-        line.model = identity
-        line.view = camera.view
-        line.projection = camera.projection
-        line.aspect = width / height
-        line.miter = 0
-        shader.uniforms.index = i / (list.length - 1)
-        line.draw()
-      })
+  function createSegment () {
+    return newArray(steps).map((i, _, list) => {
+      const x = lerp(-1, 1, i / (list.length - 1))
+      return [ x, 0, 0 ]
     })
+  }
 
-    function createSegment () {
-      return newArray(steps).map((i, _, list) => {
-        const x = lerp(-1, 1, i / (list.length - 1))
-        return [ x, 0, 0 ]
-      })
-    }
+  let analyser = null
+  setDominantColors()
 
+  function show (darkMode) {
```

Comment:
I know not your change, but generally, we prefer anonymous functions

const show = (darkMode) => { ... }

to the function notation because you don't have to worry about binding `this` ,etc. no need to change here, but wanted to call out :)